content/en/docs/v3.6/op-guide/clustering.md

```diff
@@ -228,7 +228,7 @@ Moreover, discovery URLs should ONLY be used for the initial bootstrapping of a
 
 #### Custom etcd discovery service
 
-Discovery uses an existing cluster to bootstrap itself. If using a private etcd cluster, create a URL like so:
+Discovery uses an existing cluster to bootstrap itself, note that currently it needs etcd v2 to be supported on that cluster. If using a private etcd cluster, create a URL like so:
```

Comment:
V2 discovery is deprecated in 3.6. Instead, users are recommended to use v3 discovery.

FYI. https://etcd.io/docs/v3.6/dev-internal/discovery_protocol/

This guide needs to be updated.

Reply:
I don't quite follow.  Given that it's deprecated, why are we recommending that people use it?

Reply:
Right, both v2 discovery and the public https://discovery.etcd.io/ service are deprecated. Instead, we should recommend users to use v3 discovery https://etcd.io/docs/v3.6/dev-internal/discovery_protocol/. 

I should add an example on how to use v3 discovery to bootstrap new clusters, but unfortunately I do not get time to do it so far. See https://github.com/etcd-io/website/issues/653. It would be appreciated if anyone can help to finish it.